miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -75,4 +89,17 @@ internal class MiniAppWebViewClient(private val loader: WebViewAssetLoader) : We
         view: WebView,
         request: WebResourceRequest
     ): WebResourceResponse? = loader.shouldInterceptRequest(request.url)
+
+    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    override fun onPageFinished(webView: WebView, url: String?) {
+        super.onPageFinished(webView, url)
+        try {
+            val inputStream = webView.context.assets.open("inject.js")
```

Comment:
@minh-rakuten We discussed this yesterday, let's move the file reading part to an earlier stage. Maybe the init block of `MiniAppWebViewClient`. What do you think?

Reply:
Sure. However, we still need to determine when to do an injection here so I'll update it later.

@corycaywood As I know for now, I can only injected js after the page is loaded. Could you give a support looking for a solution 🙏 ?

Reply:
I tested this with `onPageStarted` and it seems to be working. Maybe it's due to the issue I commented on below: you don't need to inject the `javascript:` keyword or use an immediately invoked function.